src/fstar/FStarC.Main.fst

```diff
@@ -113,10 +113,36 @@ let load_native_tactics () =
 (* print_in_place options are passed *)
 let fstar_files: ref (option (list string)) = Util.mk_ref None
 
-(****************************************************************************)
-(* Main function                                                            *)
-(****************************************************************************)
-let go _ =
+(* ocamlenv mode: called whenever the *first* argument is exactly 'ocamlenv' *)
+let go_ocamlenv rest_args =
+  let shellescape (s:string) : string =
+    String.list_of_string s |>
+    List.map (function
+      | '\\' -> "\\\\" // backslashes can be escaped by themselves
```

Comment:
I don't think this works:
```
❯ echo '\\'
\\
```

Reply:
Ah thanks! I misremembered, backslashes don't need escaping in single quotes.